packages/enzyme-adapter-react-helper/src/enzyme-adapter-react-install.js

```diff
@@ -0,0 +1,56 @@
+#! /usr/bin/env node
+/* eslint no-console: 0 */
+
+import shell from 'shelljs';
+import semver from 'semver';
+
+const reactEnv = process.env.REACT;
+const reactArg = process.argv[2];
+
+if (reactEnv && !semver.valid(reactEnv)) {
+  throw new Error('REACT environment variable is not a valid semver');
+}
+
+if (reactArg && !semver.valid(reactArg)) {
+  throw new Error('Argument supplied to enzyme-adapter-react-install is not a valid semver');
+}
+
+const reactVersion = reactEnv || reactArg;
+
+if (!semver.satisfies(reactVersion, '>=0.13')) {
+  throw new Error('semver is not valid. Please provide a valid semver as an argument or environment variable.');
+}
+
+console.log('Cleaning up React and related packages...');
+if (shell.exec('npm uninstall --no-save react react-dom react-test-renderer react-addons-test-utils enzyme-adapter-react-14 enzyme-adapter-react-15.4 enzyme-adapter-react-15 enzyme-adapter-react-16 && rimraf node_modules/react-test-renderer node_modules/react && npm prune').code !== 0) {
+  console.error('An uninstallation failed');
+  shell.exit(1);
+}
+
+console.log(`Installing React@${reactVersion} and related packages...`);
+if (semver.satisfies(reactVersion, '^16.0.0-0')) {
+  if (shell.exec('npm run install-peerdeps -S enzyme-adapter-react-16').code !== 0) {
```

Comment:
i think this needs to be `npm-run`; otherwise it'll look for an "install-peerdeps" script? (same throughout the file)

Reply:
altho actually, i think we need to `require('npm-run')` and use it programmatically?